doc/source/examples.rst

```diff
@@ -2,15 +2,14 @@
 Examples
```

Comment:
The general structure of the examples section is a bit confusing. 
We may want to differenciate better by moving to subpages between general example/explanation of discrimination learning, the module usage examples and minimal working examples (including data preprocessing and evaluation)